Dockerfile.import

```diff
@@ -0,0 +1,12 @@
+FROM debian:stretch
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
I think what we can do is add a `RUN apt-get update` line after the FROM line in both Dokerfile.import and Dockerfile. That way, we only need to do the update for the import container, and then we can use a cached container when doing the import for the main container.

In the line where we are doing `apt-get install` we need to keep the `apt-get update` as well (as is a Docker best practice), so we force docker to do an update whenever we change the dependencies in that line.